src/ServiceManagement/Compute/Commands.ServiceManagement/IaaS/Network/RemoveAzureReservedIP.cs

```diff
@@ -23,6 +23,15 @@ namespace Microsoft.WindowsAzure.Commands.ServiceManagement.IaaS
     [Cmdlet(VerbsCommon.Remove, ReservedIPConstants.CmdletNoun, DefaultParameterSetName = RemoveReservedIPParamSet), OutputType(typeof(ManagementOperationContext))]
     public class RemoveAzureReservedIPCmdlet : ServiceManagementBaseCmdlet
     {
+        public RemoveAzureReservedIPCmdlet()
```

Comment:
Added a parameterized constructor, so default constructor is required to support backward compatibility.